tikv-client/src/main/java/com/pingcap/tikv/Snapshot.java

```diff
@@ -76,27 +76,6 @@ public ByteString get(ByteString key) {
     return client.get(key, timestamp.getVersion());
   }
 
-  /**
-   * Issue a table read request
-   *
-   * @param dagRequest DAG request for coprocessor
-   * @return a Iterator that contains all result from this select request.
-   */
-  public Iterator<Row> tableRead(TiDAGRequest dagRequest) {
```

Comment:
tableRead is deleted since we don't use it anymore

Reply:
Will be used in https://github.com/pingcap/tispark/pull/244

Reply:
ok, I will keep it

Reply:
will IndexScanExample be used as well?

Reply:
No, `IndexScanExample.java` will not be used.